src/ImapDump/Command/AuthCommand.php

```diff
@@ -23,15 +23,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $host     = $input->getArgument('host');
         $username = $input->getArgument('username');
 
-        $question = new Question('Password: ');
-        $question->setHidden(true);
+        if (!$password = $this->checkpass($username)) {
+            $question = new Question('Password: ');
+            $question->setHidden(true);
 
-        $questionHelper = new QuestionHelper();
-        $password = $questionHelper->ask($input, $output, $question);
+            $questionHelper = new QuestionHelper();
+            $password = $questionHelper->ask($input, $output, $question);
+        }
 
         $server = new Server($host);
         $connection = $server->authenticate($username, $password);
 
         return $connection;
     }
+
+    protected function checkpass(string $username)
```

Comment:
I would suggest a more specific method name like getAccountPassword(..) or something like that.

Reply:
Thanks friend, I will send a new review with the proposed changes :)